server/odc-service/src/main/java/com/oceanbase/odc/service/iam/ResourceRoleService.java

```diff
@@ -166,6 +155,12 @@ public int deleteByResourceId(Long resourceId) {
         return userResourceRoleRepository.deleteByResourceId(resourceId);
     }
 
+    @Transactional(rollbackFor = Exception.class)
+    @SkipAuthorize("internal usage")
+    public int deleteByResourceTypeAndId(@NonNull ResourceType resourceType, @NonNull Long resourceId) {
+        return userResourceRoleRepository.deleteByResourceTypeAndId(resourceType.toString(), resourceId);
```

Comment:
Better use ResourceType instead of String.

Reply:
done